ynr/apps/candidates/models/db.py

```diff
@@ -31,6 +31,14 @@ class EditType(Enum):
     BULK_ADD = "Bulk Add"
 
 
+class ActionType(models.TextChoices):
```

Comment:
I'm not sure how to find any missing `action_type` values; I found these by browsing the code base and in the shell (although it seems that is limited to my local db).